.drone.yml

```diff
@@ -20,7 +20,7 @@ steps:
   privileged: true
 
 - name: build-all-binaries
-  image: golang:1.17.3
```

Comment:
I'd bump to 1.19 if possible

Reply:
Sounds good, even bumping to `1.18` has caused other errors in the build process, so there's likely more I have to do to get it all working properly

Reply:
bumped version to `v1.19.4`, builds are now passing